docs/authentication.md

```diff
@@ -311,3 +319,16 @@ client.exchangeToken('item_preview', 'https://api.box.com/2.0/files/123456', opt
 
 This will attach an external user name and ID to annotations made with the token,
 in order to attribute them to someone who does not have a Box account.
+
+### Revoking tokens
+
+The access tokens for a client can be revoked when needed.
```

Comment:
@mattwiller agreed. I'm thinking though, is there a different SDK api call that more directly relates to `POST /oauth2/revoke` without that side effect?

Reply:
@cbetta You could add a second code sample showing how to [revoke a token directly](http://opensource.box.com/box-node-sdk/jsdoc/BoxSDKNode.html#revokeTokens) and make that one the tagged sample.  But I think for most users, `client.revokeTokens()` will be the main way they will revoke using the SDK, so that makes sense to me to keep.

Reply:
Yes and no. I'd like to document as close as possible how to use the endpoint in the reference docs. 

Am I right to think the sample should be?

```js
sdk.revokeTokens("<TOKEN>")
  .then(() => {
    // the token has been revoked
  })
```

Reply:
@cbetta Yep, that looks right to me!